src/pages/home/sidebar/SidebarLinks.js

```diff
@@ -138,10 +138,15 @@ const SidebarLinks = (props) => {
                     >
                         <MagnifyingGlassIcon width={20} height={20} />
                     </TouchableOpacity>
-                    <AvatarWithIndicator
-                        source={props.myPersonalDetails.avatarURL}
-                        isActive={props.network && !props.network.isOffline}
-                    />
+                    <TouchableOpacity
+                        onPress={() => redirect(ROUTES.SETTINGS)}
```

Comment:
We also need to hide the sidebar when redirecting so it might be best to handle this redirect in the `HomePage`?

https://github.com/Expensify/Expensify.cash/blob/41c21c22ac7fd9a5a14f130bc714eeffae43c9e8/src/pages/home/HomePage.js#L230-L250

https://user-images.githubusercontent.com/32969087/105081719-a4875b00-5a36-11eb-8e25-fc19db278db7.mp4

Reply:
handled this in settings i think it's better that way